sites/main-site/src/content/hackathon-projects/hackathon-march-2025/gpu_testing.md

```diff
@@ -0,0 +1,36 @@
+---
+title: GPU development / testing
+category: components
+slack: "https://nfcore.slack.com/archives/C07PPQR415W"
+intro_video: ""
+image: "/assets/images/events/2025/hackathon-march/nebius_sponsor_logo.png"
+image_alt: Picture of NVIDIA H100 Graphics card
+leaders:
+  FloWuenne:
+    name: Florian Wuennemann
+    slack: "https://nfcore.slack.com/team/UU10KMQ1F"
+  alex000kim:
+    name: Alexander Kim
+    slack: "https://nfcore.slack.com/team/U08GDSB976U"
+---
+
+Thanks to the generous sponsorship from [Nebius](https://nebius.com/), we have access to 16 powerful H100 GPUs for the duration of this hackathon. Nebius is providing these resources through their Slurm-based workload manager called [Soperator](https://nebius.com/services/soperator), enabling participants to test and develop GPU-accelerated tools with Nextflow using a familiar Slurm interface.
+We welcome people from all other projects to use these resources if they need access to GPUs.
+
+To enable participants access to the Slurm cluster with the GPU resources, we need to add your public key to the authorized_keys list on the Slurm cluster. If you want access, please contact [Florian](https://nfcore.slack.com/archives/DTZKT23D1) with your public key prior to the hackathon start.
+
+You can create a dedicated ssh keypair with ssh-keygen like this (store in a safe location):
+
+`ssh-keygen -t rsa -C "your_email@example.com"`
+
+Once your public key has been added to the authorized_keys list on the cluster, you can connect to it with:
+
+`ssh -i /path/to/your/private_key username@hostname`
```

Comment:
````suggestion
```
ssh -i /path/to/your/private_key username@hostname
```
````
do we already now username@hostname?

Reply:
No, username will obviously be the persons name that provides a key and hostname will only be available once the cluster is provisioned (and might potentially change throughout the hackathon if we have to redeploy.